docs/UserGuide.md

```diff
@@ -86,25 +86,30 @@ InternConnect is a **desktop app for managing internship applicants, optimized f
 | **Job ID**  | `ji`      | 10           | Can only contain upper case alphanumeric characters and spaces |
 
 * `LOCAL_PART` can only contain alphanumeric characters and any of `+` `_` `.` `-`, but must not start or end with these special characters
-* `DOMAIN` can be made up of numerous `DOMAIN_LABEL` separated by `.`, with the last `DOMAIN_LABEL` being at least 2 characters long
-* `DOMAIN_LABEL` can only contain alphanumeric characters separated by `-`, but must start and end with alphanumeric characters
+* `DOMAIN` refers to the email domain that can be made of numerous `DOMAIN_LABEL`
+    * `DOMAIN_LABEL` can be separated by `.`
+    * `DOMAIN_LABEL` can only contain alphanumeric characters separated by `-`
```

Comment:
May I know whats the difference between line 90 and 91? It sounds contradicting

Reply:
`DOMAIN_LABEL` refers to one part of the `DOMAIN`, and each `DOMAIN_LABEL` is separated by periods. But for one `DOMAIN_LABEL` itself, should only contain alphanumeric characters and hyphens, and not started or ended by a hyphen. It is perhaps better to clarify by adding "Each" here for line 91, thanks for pointing out!